server/api/story/story.controller.js

```diff
@@ -3,7 +3,7 @@ const Story = require('./story')
 const getStory = async (req, res) => {
   try {
     const got = await Story.findOne(req.params)
-      .populate('author', 'name body imgUrl')
+      .populate('author', 'name body imgUrl') // TODO: add test that will check name and imgURL are populated
```

Comment:
if you add a TODO: in the code you must do one of the following:
a) DO the todo and remove it
b) create a Jira for the TODO so that it gets done in the future. 
c) decide it doesn't need doing and remove the TODO.